tmva/pymva/inc/TMVA/MethodPyAdaBoost.h

```diff
@@ -19,24 +19,23 @@
 //                                                                      //
 // MethodPyAdaBoost                                                     //
 //                                                                      //
-//                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
+#ifndef ROOT_TMVA_PyMethodBase
```

Comment:
What is the current coding style? Moving all the guards into the headers? What about using `#pragma once`? That would be nice ;)